queries/date.py

```diff
@@ -304,6 +305,7 @@ def help_text(lemma: str) -> str:
 
 QDateEasterSunday/fall →
     'páskadagur:kk'_et/fall
+    | "páskasunnudagur"
```

Comment:
This should probably be a lemma terminal instead of a literal terminal, but if "páskasunnudagur" is not in BÍN, you need to write out individual cases: QDateEasterSunday_nf -> "páskasunnudagur", QDateEasterSunday_þf -> "páskasunnudag", etc.